### PR TITLE
Adds links from dedication feats to archetype journal

### PR DESCRIPTION
**Pull request by xdy**
*Except* Flexible Spellcaster and Ghost which I couldn't figure out.

### CODE REVIEW

**Comment by TikaelSol** [comment]
Package lock got included

File: packs/data/feats.db/viking-dedication.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
...
+            "value": "<p>Vikings spend long periods of time at sea, only to leap from their boats at a moment's notice, charge through the surf, and take their enemies by storm. You're trained in Sailing Lore. When moving through water using your land Speed, you ignore difficult terrain resulting from shallow water, such as in bogs or on beaches. In addition, if you roll a success for your Athletics check to @Compendium[pf2e.actionspf2e.Swim]{Swim}, you get a critical success instead.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.2CbiV0VtDKZfQNte]{Viking}g archetype.</p>"
```
Comment (TikaelSol) [inline packs/data/feats.db/viking-dedication.json:16]: `{Viking}g`

The extra `g` is for the greataxe they carry.

File: packs/data/feats.db/wellspring-mage-dedication.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>There's no longer a limit on how many temporary spell slots you can gain per day from wellspring magic.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you gain two other feats from the wellspring mage archetype.</p>"
+            "value": "<p>There's no longer a limit on how many temporary spell slots you can gain per day from wellspring magic.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you gain two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.zj9z2BwVX6TLHdx3]{Wellspring Mage (Class Archetype)} archetype.</p>"
```
Comment (TikaelSol) [inline packs/data/feats.db/wellspring-mage-dedication.json:16]: Can ditch the parenthetical in the label.

File: packs/data/feats.db/mummy-dedication.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
...
... mummified, and you've risen as an undead mummy. You gain the mummy and undead traits, as well as basic undead benefits.</p>\n<p>Many effects that harm the living are far less effective against your desiccated form. Unfortunately, your dried-out body is also highly flammable. You gain the @Compendium[pf2e.feats-srd.Toughness]{Toughness} feat, but also fire weakness equal to half your level. Your fist draws moisture from the living by touch, making it more damaging and deadly. The damage die for your fist increases to 1d6 instead of 1d4, and your fist loses the nonlethal trait.</p>\n<p>Finally, you become supernaturally bound to the dominant terrain in which you were created, either because you formed naturally from the weather of that terrain or because of the local ingredients and methods used in an intentional mummification process. Choose arctic, desert, mountain, or swamp. This choice can't be changed and may alter the effects of some of your feats.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.ZewC2i5YdZPsWO8X#mummy]{Mummy} archetype.</p>"
```
Comment (TikaelSol) [inline packs/data/feats.db/mummy-dedication.json:16]: That link looks mangled, unless the UUID for mummy archetype ended up randomly ending with `#mummy` in which case I quit.

File: packs/data/feats.db/familiar-master-dedication.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
...
+            "value": "<p>You have forged a mystical bond with a creature. This might have involved complex rituals and invocations, such as meditating under the moon until something crept out of the forest. Or maybe you just did each other a good turn, such as rescuing the beast from a trap or a foe, and then being rescued in turn.</p>\n<p>Whatever the details, you are now comrades until the end. You gain a familiar. If you already have a familiar, you gain the @Compendium[pf2e.feats-srd.Enhanced Familiar]{Enhanced Familiar} feat.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.yaR8E2drX6pFUFCK]{Familiar Master} archetype.</p>"
```
Comment (TikaelSol) [inline packs/data/feats.db/familiar-master-dedication.json:16]: That UUID is testing me, right?

**Comment by TikaelSol** [comment]
Sorry about not being able to just make suggestions for the fixes, github has a bug that breaks them if they include the `@` symbol.

File: packs/data/feats.db/familiar-master-dedication.json (hunk 4 above)
Comment (xdy) [reply packs/data/feats.db/familiar-master-dedication.json:16]: Yes. :)

File: packs/data/feats.db/mummy-dedication.json (hunk 3 above)
Comment (xdy) [reply packs/data/feats.db/mummy-dedication.json:16]: Huh. Wonder how that happened?

File: packs/data/feats.db/wellspring-mage-dedication.json (hunk 2 above)
Comment (xdy) [reply packs/data/feats.db/wellspring-mage-dedication.json:16]: Check.

File: packs/data/feats.db/viking-dedication.json (hunk 1 above)
Comment (xdy) [reply packs/data/feats.db/viking-dedication.json:16]: Definitely. (Wonder how I managed to do that?)

**Pull request merged**